Beeper/BarcelonaMautrixIPC/Runtime/UnixSocketMautrixIPCChannel_Tests.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  UnixSocketMautrixIPCChannel_Tests.swift
+//  BarcelonaMautrixIPCTests
+//
+//  Created by Joonas Myhrberg on 29.1.2023.
+//
+
+import NIO
+import NIOFoundationCompat
+import XCTest
+
+@testable import BarcelonaMautrixIPC
+
+final class UnixSocketMautrixIPCChannel_Tests: XCTestCase {
```

Comment:
I'm not going to be reviewing these tests, since I don't really know how NIO works and figuring it all out may not be worth my time. I'm just trusting you here

Reply:
Yeah, it's a bit verbose. Small tip: a quick way of reviewing unit tests is to just undo the code side changes and see if the tests fail.